src/audio/general.c

```diff
@@ -1297,6 +1297,9 @@ OcarinaNote sScarecrowsLongSongSecondNote;
 #if OOT_DEBUG
 u8 sIsMalonSinging;
 f32 sMalonSingingDist;
+
+#include "debug_bss.inc.c"
+
```

Comment:
I agree with this change overall, but does this really have to be its own file? In general I find .inc.c files annoying to navigate, and the main benefit seems to be hiding large blocks of code but that's not the case here